### ISSUE TITLE
Help me pleaseee

### ISSUE THREAD

**Issue by DiogoMoreira0610** [AurelienAudero/i5-7400-EFI-by-Aure#3]
Hello Aurelien... Can you help me with my hackintosh??

I'm not sure how to make a hackintosh.... and I'm trying to make one on a PC with core i5 7400 using its EFI.... I downloaded the BigSur image through Olarila, made the pendrive through BalenaEtcher, open Explorer++ to replace EFI with it, but I can't get the installation boot to work...

Gets only this screen after Apple logo 
![WhatsApp Image 2021-06-28 at 16 48 25](https://user-images.githubusercontent.com/27218524/123695811-68e88c80-d831-11eb-8232-48a8aeac2703.jpeg)

**Comment by AurelienAudero** [comment]
Hi Diogo, 

Don't panic, i will try my best for your hackintosh to boot ans be perfect...

You used the good method for creating your USB installation key (balenaEtcher + Olarila Image, i used the exact same method)

### Questions

1) Have you followed [these steps](https://github.com/AurelienAudero/i5-7400-EFI-by-Aure#configuration) ?
2) You used the Clover or the OpenCore EFI ?
3) When you are at the Clover Screen, press the right arrow key on your keyboard or click with the mouse to get access to the settings menu (screenshot n°1 below) and go the press "Enter" key while boot-args is selected, try to add "-v" (without the quotes, like on the screenshot n°2 below)

Screenshot n°1

![958859414_Clover_r4954forVMware.png.f9a02d312c42314f16456f21ea528e71.png](https://user-images.githubusercontent.com/64486562/123698281-46b93380-d85e-11eb-816e-63d116a9b8ea.png)

Screenshot n°2

![IMG_0567.jpeg](https://user-images.githubusercontent.com/64486562/123699089-28076c80-d85f-11eb-8c4f-1d0d2ccdbafb.jpeg)

(Sorry for the quality of images, I searched for it on Google and edited it on my phone because I can't reach my PC tonight because I hurt my knee recently but I will provide you a decent help tomorrow afternoon)

_For the information about the time, I live in France 🇫🇷 and it's 22h30 or 10:30PM at the time I write this message_

Thanks for using my EFI 😉
I have a great day 👋

**Comment by DiogoMoreira0610** [comment]
Many thanks for your help... how can I change de OC to Clover? Just put the EFI with clover or need put some other BigSur image?
The OIarila image comes with OC folder inside the EFI...

**Comment by AurelienAudero** [comment]
Hi,

In the Original post you said : 
> open Explorer++ to replace EFI with it

It was the EFI Folder of the USB key or of one of your disks ?

**Comment by DiogoMoreira0610** [comment]
EFI folder of the USB

**Comment by AurelienAudero** [comment]
Hi,

---

### So to replace OC with Clover on your USB, you need to follow these steps : 
1. Mount the EFI of your USB
2. Access the root of the EFI partition with a file explorer (Explorer++ for example)
3. Delete the folder called "EFI" (at the root)
4. After deletion (⚠️ DELETION, NOT REPLACE OR MERGE ⚠️), copy the EFI Folder download on this Github repository
5. After that, you can verify that everything went well, by opening the folder called "EFI" on the EFI partition of your USB and there should be these 2 folders : 
	- BOOT
	- CLOVER
6. If there are ONLY the 2 folders above, it's GOOD ! But if there is a folder called "OC" or with another name, retry from step 3

---

Thanks again for using my EFI 😉
Have a great day 👋

**Comment by DiogoMoreira0610** [comment]
Tks for your help...
I managed to regenerate the bootavel pendrive with Monterey and followed the procedure that is in github, and the installation menu even appeared. But it crashes in 12 minutes left with the error "bless failed with error number 3". However, restarting and selecting the ssd to start the installation, it arrives in waiting less than 1 minute. Then there is an update error and it does not finish and the system is inoperative and does not boot.

Would it be possible to use this EFI Clover with BigSur 11.4?

**Comment by AurelienAudero** [comment]
Hi,

When you're at the installation screen,
1. Go to Disk Utility
2. Erase the disk where you want to install macOS and select the following parameters :
- Name : "Machintosh HD"
- Format : "APFS"
- Scheme : "GUID Partition Map"
3. After that, you can install macOS normally by selecting the "Machintosh HD" drive when prompted

---

Keep me informated
Thanks again for using my EFI 😉
Have a great day 👋

**Comment by DiogoMoreira0610** [comment]
Tks for you time helping me...

Yes, before install I erase the ssd with the options that you informed... but still get this error:
![WhatsApp Image 2021-06-30 at 09 28 42](https://user-images.githubusercontent.com/27218524/123960597-dd344480-d985-11eb-9e11-01d2ba0bed34.jpeg)
![WhatsApp Image 2021-06-30 at 09 28 49](https://user-images.githubusercontent.com/27218524/123960603-defe0800-d985-11eb-8bcf-0bf9f8f0ef5f.jpeg)

**Comment by AurelienAudero** [comment]
Did your PC has restarted while the installation goes ?

**Comment by DiogoMoreira0610** [comment]
No... occours the error and not reboot... but when restart, i choose the ssd and continue install... but dont work...

**Comment by AurelienAudero** [comment]
Hi,

I think that I finally understood what the problem is
I updated the EFI Folder and the problem should be fixed but keep me informed about the situation

You can download the new version of this EFI at the link below : 
https://github.com/AurelienAudero/i5-7400-EFI-by-Aure/releases/tag/r5137-v2

For installing you need to : 
1. Follow [these steps](https://github.com/AurelienAudero/i5-7400-EFI-by-Aure#configuration) if it you didn't
2. Boot on Clover and select "Boot macOS Install from USB"
3. Follow the steps format disk using Disk Utility with the following parameters : 
	- Name : "Macintosh HD" (It's important to have the exact same name because it can be issue when updating macOS in the future, For information "HD" mean Hard Drive, so Macintosh Hard Drive, I tell you about that because I saw in the screenshot of a post above you called it "Macintosh SSD")
	- Format : "APFS"
	- Scheme : "GUID Partition Map"
4. Quit Disk Utility and proceed to installation normally by selecting "Macintosh HD"
5. After rebooting, access to Boot Menu (by pressing F11 on MSI Motherboard) and boot to Clover again
6. Select "Boot macOS Install from Macintosh HD via Preboot"
7. It's will load and finish the install
8. It will restart multiples times, so at each restart, repeat the steps 5, 6 and 7 (of this post)
9. You will arrive to the macOS Setup screen and Congratulations 🎉
10. You can now setup macOS as you would on a normal Mac

---

Keep me informated
Thanks again for using my EFI 😉
Have a great day 👋

**Comment by DiogoMoreira0610** [comment]
Tks, Aurelien.... I tried this new EFI,  and I followed the steps you mentioned, but the same problem occurred.

**Comment by DiogoMoreira0610** [comment]
I deleted the EFI folder of pendrive and copied the new...

**Comment by AurelienAudero** [comment]
Sorry but I'm out of ideas for the moments.

### Questions :
1. Did you try to re-make the USB Installation Key ? (Go to balenaEtcher and flash the USB key again)
2. **If the error occurs after that**, please follow these steps : 
	- Copy ALL the logs ("CMD/Win + A" and "CMD/Win + C")
	- Go to https://pastebin.com
	- Paste the logs in the Red Rectangle of the screenshot below
	- Choose a title where it's mentioned (info in the screenshot below too)
	- Click on the "Create New Paste" button

---

### Screenshot : 
![Capture d’écran 2021-06-30 à 19 51 32](https://user-images.githubusercontent.com/64486562/124009101-7b9fc480-d9dd-11eb-89be-21d627c30c1b.png)

---

Keep me informated
Thanks again for using my EFI 😉
Have a great day 👋

**Comment by AurelienAudero** [comment]
Is your problem has been solved ?

**Comment by DiogoMoreira0610** [comment]
Hello Aurelien... 
I had surgery these days and I haven't been able to test it before... this week I should do a new test. 

Thanks for your help....

**Comment by AurelienAudero** [comment]
Hi,
No problem, take care of yourself
And keep me informated

---

Thanks again for using my EFI 😉
Have a great day 👋

**Comment by DiogoMoreira0610** [comment]
Aurelien, have a great day....

This new EFI work with BigSur 11.4 or only with Monterey?

**Comment by AurelienAudero** [comment]
I only tested it with Monterey
So I don't know if it's works with Big Sur
I may not work but you can try and keep me informated about what's happening

---

Thanks again for using my EFI 😉
Have a great day 👋

**Comment by DiogoMoreira0610** [comment]
Hello Aurelien... Finally i got a hackintosh thats work..... i install the bigSur with opencore default EFI generate with iMac18,1.... 

Thansk for your help....

**Comment by AurelienAudero** [comment]
Hi,

Sorry for my late answer.
With the information you gave me, i will mark this EFI as incompatible with Big Sur.
When you will upgrade to Monterey, don't forget to give a look here.

Can I close this issue ?
Do you any other questions ?

---

Thanks for your patience and all the informations 😉
Have a great day and take care of yourself 👋
